src/stores/attributes-returns.md

```diff
@@ -61,7 +61,13 @@ _Returns Attributes_
 
 1. In the left panel, choose **Manage Labels/Options**.
 
-1. Enter the **Default Store View**.
+1. In the **Manage Titles (Size, Color, etc.)** section, enter the title value for each Store View.
+
+{:.bs-callout-info}
+If the **Input Type** is `Dropdown` you can manage options in **Manage Options (Values of Your Attribute)** section.
+Click <span class="btn">Add Option</span> for adding new option, enter the title for Admin and each Store View.
+Click **Is Default** radiobutton, to make an option selected by default.
+Click **Delete** to delete an option.
 
 1. To save changes, click <span class="btn">Save Attribute</span>.
```

Comment:
Let's replace the screen capture (replace the file using the same file name) with one that shows the options as described:

![image](https://user-images.githubusercontent.com/1128913/156060842-6395c6cc-a533-47d4-84ac-ef87a85deb7e.png)